docs/unlocking_the_bootloader.md

```diff
@@ -0,0 +1,35 @@
+#### On unlocking the bootloader
+
+First of all, this tool will not help you bypass any vender locked bootloaders in non-official ways.
+
+Devices by *Samsung, Google, OnePlus, Fairphone* and others make it relatively easy to unlock the bootloader and get good support in the installer.
+
+For some devices, it is necessary to unlock the bootloader manually. You usually need to create an account on the manufacturer's website, wait for a certain time and obtain a code from there. OpenAndroidInstaller will try to guide you as far as possible. These vendors include *Sony, Motorola and Xiaomi* amongst others.
+
+Other phone manufacturers no longer allow you to unlock the bootloader. Nothing can be done if you have not unlocked your device in time. These manufacturers include *Huawei, Honor, LG and ASUS*. Support for these manufacturers will always be very limited.
+
+<details>
+    <summary>
+        <b>Here is a brief overview of the bootloader policies of some of the most popular brands (may be incomplete/inaccurate)</b>
+    </summary>
```

Comment:
The details here could be excluded, it would be more clear to see the instructions that way

Reply:
Yes, but it would also bloat the page, as the fully detailed list is already linked.

Reply:
It is not in English, therefore being less accessible, I recommend you reconsider that

Reply:
Yeah, I also don't like, that its not in English... Maybe just showing the table would be better.

Reply:
> Yeah, I also don't like, that its not in English... Maybe just showing the table would be better.

hmmm ok - can you please fix that quickly, as I can't commit to dev directly

Reply:
done